.github/workflows/sgl.yml

```diff
@@ -0,0 +1,66 @@
+name: sgl
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  pull_request:
```

Comment:
we recently added configs for both pull_request and push to make regression monitoring more reliable. please make a copy and add push. Thank you!